roles/openshift_sanitize_inventory/tasks/__logging.yml

```diff
@@ -0,0 +1,25 @@
+---
+# this is used to set the logging variables from deprecated values to the current variables names
+# this file should be deleted once variables are no longer honored
+
+- set_fact:
+    openshift_logging_install_logging: openshift_hosted_logging_deploy | default(false) | bool
+    logging_hostname: "{{ openshift_hosted_logging_hostname | default('kibana.' ~ (openshift_master_default_subdomain | default('router.default.svc.cluster.local', true))) }}"
+    logging_ops_hostname: "{{ openshift_hosted_logging_ops_hostname | default('kibana-ops.' ~ (openshift_master_default_subdomain | default('router.default.svc.cluster.local', true))) }}"
+    logging_elasticsearch_cluster_size: "{{ openshift_hosted_logging_elasticsearch_cluster_size | default(1) }}"
+    logging_elasticsearch_ops_cluster_size: "{{ openshift_hosted_logging_elasticsearch_ops_cluster_size | default(1) }}"
+    openshift_logging_kibana_hostname: "{{ openshift_hosted_logging_hostname | default('kibana.' ~ (openshift_master_default_subdomain | default('router.default.svc.cluster.local', true))) }}"
+
+    openshift_logging_storage_kind: "{{ openshift_hosted_logging_storage_kind }}"
```

Comment:
I need to go through these a bit further, there starts to be issues decoupling this from the generation of NFS PVs within other roles that I'm unsure of how to approach. Ideally variables like this would just have defaults within the `openshift_logging` role, so I would end up removing this fact from this task file...